onto-viewer-core/src/main/java/org/edmcouncil/spec/ontoviewer/core/ontology/data/label/provider/LabelProvider.java

```diff
@@ -172,18 +189,18 @@ private void labelProcessing(String lang, Map<String, String> labels, String lab
 
         labels.put(label, lang);
         LOG.debug("[Label Extractor]: Extract label: '{}' @ '{}' for element with IRI: '{}'",
-                label, lang.isEmpty() ? "no-lang" : lang, entityIri.toString());
+            label, lang.isEmpty() ? "no-lang" : lang, entityIri.toString());
```

Comment:
You don't need to call `toString()` on `entityIri` as it will be done by a logger. So, use:
```
    label, lang.isEmpty() ? "no-lang" : lang, entityIri);
```